README.md

```diff
@@ -10,13 +10,30 @@ A suite of pre-built Dojo 2 widgets, ready to use in your application. These wid
 
 **WARNING** This is *alpha* software. It is not yet production ready, so you should use at your own risk.
 
+- [Usage](#usage)
 - [Features](#features)
-- [How Do I use this package?](#how-do-i-use-this-package)
+- [Conventions](#conventions)
+  - [Icons](#icons)
+  - [Coding conventions](#coding-conventions)
 - [How Do I Contribute?](#how-do-i-contribute)
     - [Installation](#installation)
     - [Testing](#testing)
 - [Licensing Information](#licensing-information)
 
+## Usage
+
+To use `@dojo/widgets`, install the package:
+
+```bash
+npm install @dojo/widgets
+```
+
+You can also import each widget individually, e.g. for our `ComboBox`:
```

Comment:
Actually we probably shouldn't include this in the readme for the moment as it's not planned until beta 2